db/schema.rb

```diff
@@ -131,6 +131,8 @@
     t.integer "eligible_itt_subject"
     t.boolean "teaching_subject_now"
     t.string "itt_academic_year", limit: 9
+    t.boolean "postgraduate_masters_loan"
+    t.boolean "postgraduate_doctoral_loan"
```

Comment:
@nomasprime These two attributes have been moved to the claim model. Could you please fix

Reply:
Missed this. Don't know why it keeps coming back.

I'll remove again.